test/known_issues/known_issues.status

```diff
@@ -5,6 +5,7 @@ prefix known_issues
 # sample-test                        : SKIP
 
 [true] # This section applies to all platforms
+test-repl-uncaught-exception-error: SKIP
```

Comment:
Why is this skipped everywhere? If it passes then it shouldn't be in known issues.

Reply:
Please check the comments inside of the test (https://github.com/nodejs/node/pull/20803/files#diff-4f5313e8a4583fdc4637c1a56958e45fR7).

Thrown errors are suppressed even though they should not be. I could not think of a way to write the test that it would fail and still show the mentioned issue.

Reply:
But marking the test skipped everywhere means that this is dead code that isn't executed.

Reply:
I know... :/ do you have a suggestion what to do instead? I would really like to have a known issue test for this. IMO this is a pretty nasty error.

Reply:
I could move the test to `parallel`. It would be an `inverse` known issue... It would fail in case we fix the issue.

Reply:
> 
> 
> I know... :/ do you have a suggestion what to do instead? I would really like to have a known issue test for this. IMO this is a pretty nasty error.

cc @nodejs/testing

Reply:
I fixed it by using a child process. That way I verify if the process kept on running after the exception or not.

It will now always fail since it keeps on running.